documentation/source/quickstart.rst

```diff
@@ -87,22 +87,22 @@ test.
 
 .. code-block:: bash
 
-    $> export PATH=/opt/pym32/bin
-    $> cd <cocotb_dir>
+    $> export PATH=/opt/pym32/bin:$PATH
+    $> cd `cocotb-config --prefix`
     $> ARCH=i686 make
```

Comment:
Two comments here:
- That whole 32 bit Linux section is a bit out of place. It's probably only needed for the free version of ModelSim? Can we move it to a more modelsim-specific page? I don't really want to start the "quickstart" documentation by a obscure edge case/workaround.
- The example how to use cocotb is out of place (cocotb is only installed further down in the documentation), and not really accurate either.

Reply:
Maybe we could move that 32-bit section to https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions and just add a link in the docs.

I'll try to clean up the rest a bit while trying to resist the urge to restructure anything.
A restructuring should be done in a different issue/PR, I'd like to follow https://www.divio.com/blog/documentation/ for that.

Reply:
Moving to tier 2: sounds like a good plan.

I agree with you, all other things can come later.

Reply:
I have re-arranged Quickstart a bit, please review.

Someone with Windows experience should improve this section - is e.g. Msys 1.0 still correct? And I guess using Anaconda/Miniconda would be the way to go these days? @themperek

Reply:
The diff is hard to read, here is a rendered version: https://cocotb-cmarqu.readthedocs.io/en/fix-fixme-docstrings/quickstart.html